File: app/src/main/java/com/example/test28/MainActivity.java
package com.example.test28;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.util.Log;
import android.widget.EditText;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Intent intent = getIntent();
        String str  = intent.getExtras().getString("키");
        //EditText edittext  = findViewById(R.id.edit_text);

        ((EditText) findViewById(R.id.edit_text)).setText(str);
        Log.d("로그", str);

            // o- cmmit

                //o-master



    }




}

File: test21/app/src/main/java/com/example/test21/MainActivity.kt
package com.example.test21

import android.content.DialogInterface
import android.content.Intent
import android.os.Bundle
import android.widget.EditText
import androidx.appcompat.app.AlertDialog
import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_main.*
import kotlinx.android.synthetic.main.flag1.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        var render = Render(this)
        render.setAnimation(Bounce().InDown(btn))
        render.start()

    }
}